js/src/helpers/addPreview.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
```

Comment:
The Travis builds are failing because the package [headless-block-components](https://github.com/kienstra/headless-block-components) isn't published yet, it's only available locally.